configure

```diff
@@ -797,6 +798,25 @@ read_registry_file()
 	done
 }
 
+substitute_words()
+{
+	local word new_words list
```

Comment:
Shouldn't `newlist` also be a local?

Reply:
newlist should be local from programming specification, it is careless of me.